packages/iov-core/src/multichainsigner.spec.ts

```diff
@@ -87,7 +85,8 @@ describe("MultiChainSigner", () => {
           tokenTicker: cash,
         },
       };
-      await signer.signAndCommit(sendTx, mainWalletId);
+      const postResponse = await signer.signAndCommit(sendTx, mainWalletId);
```

Comment:
from `SignAndCommit` to `SignAndCommitAndWaitInBlockConfirmation` will be an accurate function name description change

Reply:
same as above: "signAndCommit" does not wait for a block. Only when you subscribe to blockInfo, you will get updates about the block. This is the reason we need to wait for the block separatly one line below.